README.md

```diff
@@ -2,17 +2,21 @@ Dropwizard ActiveMQ Bundle
 ==================================
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.kjetland.dropwizard/dropwizard-activemq/badge.svg)](http://search.maven.org/#search%7Cga%7C1%7Cdropwizard-activemq)
 
-*Since Dropwizard ActiveMQ Bundle is written in Java 8, your app must also be compiled with Java 8 (But you can of course still use Dropwizard which is not Java 8).*
+*Since Dropwizard ActiveMQ Bundle is written in Java 8, your app must also be compiled with Java 8*
 
-Use it when you need to send and receive JSON (jackson) via ActiveMq in your Dropwizard 0.7.1 application.
+Use it when you need to send and receive JSON (jackson) via ActiveMq in your Dropwizard application.
 
 Please have a look at the [Example application](https://github.com/mbknor/dropwizard-activemq-bundle/tree/master/example).
 
 Change History
--------
-
+--------------
 Current version is: **0.4.0**
 
+Version 0.5.0
+
+* Add support to connect to multiple activeMq brokers
+* Upgraded to Dropwizard 1.0.2 and ActiveMQ 5.14.1
+-

```

Comment:
I think you should mention ActiveMQMultiBundle down in the "code" section where we show how  to use it,